src/api_routes_http.rs

```diff
@@ -120,7 +121,8 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("", web::get().to(get_site))
           // Admin Actions
           .route("", web::post().to(create_site))
-          .route("", web::put().to(update_site)),
+          .route("", web::put().to(update_site))
+          .service(web::resource("/block").route(web::post().to(block_instance))),
```

Comment:
Why does this need a service? Why not `.route("/block")` like some of those below?

Reply:
fixed